Source: https://bitbucket.org/sshguard/sshguard/commits/04e656373cba3100f68e33ddccd6be448a6328da
Upstream: yes
Reason: fix cross compilation

From 04e656373cba3100f68e33ddccd6be448a6328da Mon Sep 17 00:00:00 2001
From: Steven Siloti <ssiloti@gmail.com>
Date: Sat, 20 Feb 2016 09:09:38 -0800
Subject: [PATCH] Add missing AM_PROG_AR

Submitted by:   Steven Siloti (pull request #9)
---
 configure.ac | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configure.ac b/configure.ac
index e769231..bc69fba 100644
--- a/configure.ac
+++ b/configure.ac
@@ -131,6 +131,7 @@ AC_PROG_CC_C99
 AC_PROG_EGREP
 AC_PROG_RANLIB
 AC_PROG_YACC
+AM_PROG_AR
 AM_PROG_LEX
 
 ##############################################################################
-- 
2.1.1.1.g1fb337f

